tests/unit/Types/ContextFactoryTest.php

```diff
@@ -88,6 +88,26 @@ public function testReadsAliasesFromProvidedNamespaceAndContent()
 
             $this->assertSame($expected, $context->getNamespaceAliases());
         }
+
+        public function testTraitUseIsNotDetectedAsNamespaceUse()
```

Comment:
I am missing an `@covers` statement with this test; meaning that coverage will potentially show a skewed number of lines that were covered